mpisppy/utils/sputils.py

```diff
@@ -508,8 +507,8 @@ def ef_nonants(ef):
     Note:
         not on an EF object because not all ef's are part of an EF object
     """
-    for (ndn,i), var in ef.ref_vars.items():
-        yield (ndn, var, pyo.value(var))
+    for key, val in ef.ref_vars.items():
+        yield (key[0], val, pyo.value(val))
```

Comment:
I preferred this as-it-was, and it is a non-functioning change.

Reply:
Done